src/zorak/cogs/general/general_suggest.py

```diff
@@ -27,12 +27,25 @@ async def suggest(self, ctx, question):
         logger.info("%s used the %s command."
                     , ctx.author.name
                     , ctx.command)
-
-        embed = discord.Embed(description=question)
-        embed.set_author(name=f"Suggestion by {ctx.author.name}")
-        msg = await ctx.send(embed=embed)
-        await msg.add_reaction("👍")
-        await msg.add_reaction("👎")
+        
+        '''
+        
+        '''
+        embed = discord.Embed(description= question)
+        embed.set_author(name= f"Suggestion by user {ctx.author.name}")
+        error_embed = discord.Embed(title= "**Oops...**", description= "Slow down, please only use /suggest in #📎suggestions!", color= discord.Color.red())
+
+
+        '''
+        If the channel name/ID of the command matches with the current 
+        channel, the suggestion will be posted. If not, an error message will occur.
+        '''
+        if ctx.channel.name == "📎suggestions" or ctx.channel_id == "962415552737996800":
```

Comment:
Here too, generally, we bring "guild specific" things from the server_settings using 

`self.bot.server_settings.log_channel["suggestions"]`

Then we toss the Channel ID into Resources/ServerConfig/PracticalPython/channel_info.toml

To make the TOML file and the "bot" work together, we have a server_settings class that the entry must be added to as well at src/zorak/utilities/core/server_settings.py